Scripts/RemediationScripts/Readme.md

```diff
@@ -1,4 +1,54 @@
+|Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities|Remove external accounts from Azure subscriptions|[Remediate-DontUseNonADIdentities](Remediate-DontUseNonADIdentities.ps1)|Owner/User Access Administrator at subscription level|Restricted to 'User' account type.|Yes|
+|Azure_Subscription_AuthZ_Limit_ClassicAdmin_Count|Limit access per subscription to 2 or less classic administrators|[Remediate-ClassicAdminRoleAssignment](Remediate-ClassicAdminRoleAssignment.ps1)|Owner/User Access Administrator at subscription level|Restricted to 'User' account type.|Yes|
+|Azure_Subscription_AuthZ_Remove_Deprecated_Accounts|Remove Orphaned accounts from your subscription(s)|[Remediate-InvalidAADObjectRoleAssignments](Remediate-InvalidAADObjectRoleAssignments.ps1)|Owner/User Access Administrator at subscription level|Restricted to 'User' account type.|No|
+|Azure_Subscription_AuthZ_Remove_Management_Certs|Do not use management certificates|[Remediate-DoNotUseManagementCertificates](Remediate-DoNotUseManagementCertificates.ps1)|ServiceAdministrator/CoAdministrator|Restricted to 'User' account type.|No|
+|Azure_Subscription_Config_ASC_Defender|Enable all Azure Defender plans in Azure Security Center|[Remediate-ConfigAzureDefender](Remediate-ConfigAzureDefender.ps1)|Owner/Contributor at subscription level|Restricted to 'User' account type.|Yes|
+|Azure_Subscription_Configure_Conditional_Access_for_PIM|Enable policy to require PIM elevation from SAW for admin roles in Azure subscriptions|[Remediate-ConfigureConditionalAccessPolicyForPIM](Remediate-ConfigureConditionalAccessPolicyForPIM.ps1)|Owner/User Access Administrator at subscription level|Restricted to 'User' account type.|Yes|
+|1. Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access </br>2. Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access_RG </br>3. Azure_Subscription_Use_Only_Alt_Credentials|1. Do not grant permanent access for privileged subscription level roles </br>2. Do not grant permanent access for privileged roles at resource group level </br>3. Use Smart-Card ALT (SC-ALT) accounts to access critical roles on subscription and resource groups|[Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments](/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments.ps1)|Owner/User Access Administrator at subscription level|Restricted to 'User' account type.|No|
+
+</br>
+** Both System assigned and User assigned managed identities are supported. 
+* To connect Azure Account using **System assigned managed identity**, use following command:</br>
+  Connect-AzAccount -Identity
+
+* To connect Azure Account using **User assigned managed identity**, use following command:</br>
+  Connect-AzAccount -Identity -AccountId 'object-id'
+</br>
+
+## **How to use BRS**
```

Comment:
Pls. expand the BRS shortform with **(BRS)** at the end.

Reply:
Done.